watchtower/wtclient/candidate_iterator.go

```diff
@@ -0,0 +1,82 @@
+package wtclient
+
+import (
+	"container/list"
+	"sync"
+
+	"github.com/lightningnetwork/lnd/watchtower/wtdb"
+)
+
+// TowerCandidateIterator provides an abstraction for iterating through possible
```

Comment:
yes my thinking here is that this could later be backed by a graph iterator in the public case. we can also expand this tower iterator to be mutable in the case that we want the set to be modified via rpc after cli configuration